src/main/groovy/wooga/gradle/unity/tasks/Test.groovy

```diff
@@ -31,8 +36,16 @@ abstract class Test extends UnityTask implements UnityTestSpec {
         reports = value
     }
 
+    private final ConfigurableFileCollection inputFiles
+
+    @InputFiles
+    ConfigurableFileCollection getInputFiles() {
+        inputFiles
+    }
+
     @Inject
     Test() {
+        inputFiles = project.objects.fileCollection()
```

Comment:
These should be declared in the `UnityTestSpec` trait

Reply:
and Done.